### PR TITLE
Added All Controls to Control guide

### PR DESCRIPTION
**Pull request by Kurosaki077**
All controls added to the control guide.

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -5,6 +5,26 @@
 </head>
 <body>
     <h1>Xbox controller panel</h1>
+   <ul>
+    <li><b>Left Trigger</b>:Move Backward</li>
+    <li><b>Left Bumper</b>:Belt</li>
+    <li><b>L3</b>:Preset 1</li>
```
Comment (BigBroccoli) [inline website/index.html:11]: Maybe make this "Camera Preset 1" so it's a little more descriptive

File: website/index.html [hunk 2]
```diff
@@ -5,6 +5,26 @@
...
+    <li><b>Right Trigger</b>:Move Forward</li>
+    <li><b>Right Bumper</b>:Move Forward Slowly</li>
+    <li><b>R3</b>:Preset 2</li>
```
Comment (BigBroccoli) [inline website/index.html:14]: "Camera Preset 2"

File: website/index.html [hunk 3]
```diff
@@ -5,6 +5,26 @@
...
+    <li><b>Select</b>:Start Dig Autonomy</li>
+    <li><b>Start</b>:Preset 3li>
```
Comment (BigBroccoli) [inline website/index.html:16]: "Camera Preset 3" 
Also there looks to be a typo: <li><b>Start</b>:Preset 3</li>

File: website/index.html [hunk 4]
```diff
@@ -5,6 +5,26 @@
...
+    <li><b>dUP</b>:Raise belt</li>
+    <li><b>dDOWN</b>:Lower belt</li>
+    <li><b>dLeft</b>:Raise camera</li>
+    <li><b>dRight</b>:Lower camera</li>
+    <li><b>Y</b>:Belt Speed</li>
+    <li><b>X</b>:Deposit Button</li>
```
Comment (BigBroccoli) [inline website/index.html:22]: Maybe just make this "Deposit" or "Auger"

**Review by BigBroccoli — CHANGES_REQUESTED** [review]
A few minor things just with wording. Can we also get a screenshot to how this looks on the site?

File: website/index.html (hunk 1 above)
Comment (Kurosaki077) [reply website/index.html]: image
Image of completed control guide.

File: website/index.html
Comment (abarnes6) [inline website/index.html:1]: I would try to make the list wrap to the right, so it doesn't take up so much vertical space. You can probably do this with a div and some css.

**Review by abarnes6 — CHANGES_REQUESTED** [review]
Looking good, just a couple of changes and it should be good to go.

File: website/index.html
Comment (BigBroccoli) [reply website/index.html:1]: Either do this or have the camera feed to the right of the list

**Review by BigBroccoli — APPROVED** [review]
Looks fancy

**Review by abarnes6 — APPROVED** [review]

**Pull request merged**